From ac6d1aee2d79103f538a57df7253ecca339a35eb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Cristian=20One=C8=9B?= <onet.cristian@gmail.com>
Date: Sat, 26 Jan 2013 11:36:06 +0200
Subject: [PATCH 19/22] Fix the category selection actions in the find
 transactions dialog. Call 'selectAllSubItems' only for the root category
 items.

BUG: 313874
(cherry picked from commit 7a09b9a12368e39a7a1958989e36f5356e9b6278)
---
 kmymoney/widgets/kmymoneyaccountselector.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kmymoney/widgets/kmymoneyaccountselector.cpp b/kmymoney/widgets/kmymoneyaccountselector.cpp
index 6e962c5..61a1120 100644
--- a/kmymoney/widgets/kmymoneyaccountselector.cpp
+++ b/kmymoney/widgets/kmymoneyaccountselector.cpp
@@ -111,7 +111,7 @@ void kMyMoneyAccountSelector::selectCategories(const bool income, const bool exp
   for (; *it_v != 0; ++it_v) {
     if ((*it_v)->text(0) == i18n("Income categories"))
       selectAllSubItems(*it_v, income);
-    else
+    else if ((*it_v)->text(0) == i18n("Expense categories"))
       selectAllSubItems(*it_v, expense);
   }
   emit stateChanged();
-- 
1.8.1.4

